cradle-core/src/main/java/com/exactpro/cradle/testevents/TestEventBatchToStore.java

```diff
@@ -59,14 +60,14 @@ public static TestEventBatchToStoreBuilder builder(int maxBatchSize)
 		return new TestEventBatchToStoreBuilder(maxBatchSize);
 	}
 	
-	
+
 	@Override
 	public Set<StoredMessageId> getMessages()
 	{
 		if (messages == null)  //This is the case when validateTestEvent() is called from super constructor
```

Comment:
messages is never null

Reply:
There is a comment with explanation why this checking is presented here.:
`This is the case when validateTestEvent() is called from super constructor`